app/assets/stylesheets/application/modules/_checkboxes.scss

```diff
@@ -0,0 +1,74 @@
+.mu-radio {
+  div {
+    clear: both;
+    overflow: hidden;
+  }
+  label {
+    width: 100%;
+    font-weight: normal;
+  }
+  input[type="radio"],
+  input[type="checkbox"] {
+    &:empty {
+      display: none;
+      ~ label {
+        position: relative;
+        line-height: 2em;
+        text-indent: 2.5em;
+        cursor: pointer;
+        user-select: none;
+
+        &:before {
+          position: absolute;
+          display: block;
+          top: 0;
+          bottom: 0;
+          left: 0;
+          content: '';
+          width: 2em;
+          background: #D1D3D4;
+          border-radius: 3px 0 0 3px;
+        }
+      }
+    }
+    &:hover:not(:checked) ~ label {
+      color: #888;
+      &:before {
+        content: '\2714';
+        text-indent: .9em;
+        color: #C2C2C2;
+      }
+    }
+    &:checked ~ label {
+      color: #777;
+
+      &:before {
+        content: '\2714';
+        text-indent: .6em;
+        color: #333;
+        background-color: #ccc;
+      }
+    }
+    &:focus ~ label:before {
+      box-shadow: 0 0 0 3px #999;
+    }
+  }
+  &-default {
+    input[type="radio"] {
+      &:checked ~ label:before {
+        color: white;
+        background-color: $brand-complementary;
+        border-radius: 1em;
+      }
+      &:not(:checked) ~ label:before {
+        border-radius: 1em;
+      }
+    }
+    input[type="checkbox"] {
+      &:checked ~ label:before {
```

Comment:
Have you added those styles to `mumuki-styles`, right?

If that is true, please don't repeat the style logic here, and use the `mumuki-styles` bower component instead. Use [rails-assets](https://rails-assets.org/), in the same way as we used it to include the `devawesome` font.